napari/_vendor/darkdetect/__init__.py

```diff
@@ -0,0 +1,32 @@
+#-----------------------------------------------------------------------------
+#  Copyright (C) 2019 Alberto Sottile
+#
+#  Distributed under the terms of the 3-clause BSD License.
+#-----------------------------------------------------------------------------
+
+__version__ = '0.5.0'
+
+import sys
+import platform
+
+if sys.platform == "darwin":
+    from distutils.version import LooseVersion as V
+    if V(platform.mac_ver()[0]) < V("10.14"):
+        from ._dummy import *
+    else:
+        from ._mac_detect import *
+    del V
+elif sys.platform == "win32" and platform.release() == "10":
```

Comment:
(now I wonder how many features windows 11 users like me are going to miss 😅 )
I hope upstream would be able to patch this asap

Reply:
The dev was very responsive for the macOS fix, so I think it should be no problem, just open an issue.

Reply:
Asked about this! 

https://github.com/albertosottile/darkdetect/issues/13

Reply:
I believe the internal version number for windows 11 is still 10 ...

for example [this](https://www.anoopcnair.com/windows-11-version-numbers-build-numbers-major/) says:

 > **Windows 11** Major Minor Build Rev Details
Let’s understand Windows 11 major, minor, build, and rev in a bit more detail. You need to understand these types of version details from the below table. In the below table, I have taken an example of **Windows 11** insider build number **10**.0.22000.51

(Emphasis mine)